python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -50,22 +50,28 @@ def lazyLoadJobApplication(this_job):
     return lazyLoadJobObject(this_job, 'application')
 
 
-def lazyLoadJobObject(raw_job, this_attr):
+def lazyLoadJobObject(raw_job, this_attr, dont_eval=False):
```

Comment:
True, it's a quick hack though I was very confused by the code otherwise trying to evaluate 'completed' and 'new' into objects quite a lot